README.md

```diff
@@ -229,10 +244,29 @@ me@somewhere:haskell-backend$
 [haskell-language-server]: https://github.com/haskell/haskell-language-server
 [language server]: https://langserver.org/
 [hlint]: https://github.com/ndmitchell/hlint
+[fourmolu]: https://github.com/fourmolu/fourmolu
 [runtimeverification.cachix.org]: https://runtimeverification.cachix.org/
 [Nix]: https://nixos.org
 [entr]: https://github.com/eradman/entr
 [fd]: https://github.com/sharkdp/fd
 [kup]: https://github.com/runtimeverification/k/releases/latest 
 [ghcup]: https://www.haskell.org/ghcup/
 [Haskell extension]: https://marketplace.visualstudio.com/items?itemName=haskell.haskell
+[KORE RPC protocol]: ./docs/2022-07-18-JSON-RPC-Server-API.md
+[pyk]: https://github.com/runtimeverification/k/tree/master/pyk
+
+
+### Generating an integration test from a bug-report.tar.gz
+
+1) Rename `bug-report.tar.gz` to something more descriptive like `issue-123.tar.gz`
+2) Copy the tar file into `test/rpc-integration/`
+3) Run `./generateDirectoryTest.sh issue-123.tar.gz`. This will copy the definition files into `resources/` and rpc commands into `test-issue-123/`
+4) Run the test via `./runDirectoryTest test-issue-123`
```

Comment:
BTW it is possible to run a test directly from a tarball with `runDirectoryTest.sh`. We are just not doing that because updating the tests would be cumbersome for larger tarballs.